```
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/private/var/folders/y6/nj790rtn62lfktb1sh__79hc0000gn/T/cibw-run-uqf9ogxo/cp39-macosx_x86_64/venv-test-x86_64/lib/python3.9/site-packages/coincurve/__init__.py", line 1, in <module>
      from coincurve.context import GLOBAL_CONTEXT, Context
    File "/private/var/folders/y6/nj790rtn62lfktb1sh__79hc0000gn/T/cibw-run-uqf9ogxo/cp39-macosx_x86_64/venv-test-x86_64/lib/python3.9/site-packages/coincurve/context.py", line 6, in <module>
      from coincurve._libsecp256k1 import ffi, lib
  ImportError: dlopen(/private/var/folders/y6/nj790rtn62lfktb1sh__79hc0000gn/T/cibw-run-uqf9ogxo/cp39-macosx_x86_64/venv-test-x86_64/lib/python3.9/site-packages/coincurve/_cffi_backend.cpython-39-darwin.so, 0x0002): tried: '/private/var/folders/y6/nj790rtn62lfktb1sh__79hc0000gn/T/cibw-run-uqf9ogxo/cp39-macosx_x86_64/venv-test-x86_64/lib/python3.9/site-packages/coincurve/_cffi_backend.cpython-39-darwin.so' (mach-o file, but is an incompatible architecture (have 'arm64', need 'x86_64')), '/System/Volumes/Preboot/Cryptexes/OS/private/var/folders/y6/nj790rtn62lfktb1sh__79hc0000gn/T/cibw-run-uqf9ogxo/cp39-macosx_x86_64/venv-test-x86_64/lib/python3.9/site-packages/coincurve/_cffi_backend.cpython-39-darwin.so' (no such file), '/private/var/folders/y6/nj790rtn62lfktb1sh__79hc0000gn/T/cibw-run-uqf9ogxo/cp39-macosx_x86_64/venv-test-x86_64/lib/python3.9/site-packages/coincurve/_cffi_backend.cpython-39-darwin.so' (mach-o file, but is an incompatible architecture (have 'arm64', need 'x86_64'))
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -140,7 +140,6 @@ jobs:
       uses: pypa/cibuildwheel@v2.23
       env:
         CIBW_ARCHS_MACOS: x86_64
-        MACOSX_DEPLOYMENT_TARGET: 10.13
 
     - uses: actions/upload-artifact@v4
       with:

--- a/.github/workflows/verify_shared_build.yml
+++ b/.github/workflows/verify_shared_build.yml
@@ -87,7 +87,7 @@ jobs:
     name: Build macOS wheels
     needs:
     - test
-    runs-on: macos-13
+    runs-on: macos-14
 
     steps:
     - uses: actions/checkout@v4

--- a/.github/workflows/verify_shared_build.yml
+++ b/.github/workflows/verify_shared_build.yml
@@ -96,7 +96,7 @@ jobs:
       uses: pypa/cibuildwheel@v2.23
       env:
         CIBW_ARCHS_MACOS: x86_64
-        MACOSX_DEPLOYEMENT_TARGET: 10.13
+        MACOSX_DEPLOYMENT_TARGET: 10.13
 
   macos-wheels-arm:
     name: Build macOS wheels for ARM64

--- a/.github/workflows/verify_shared_build.yml
+++ b/.github/workflows/verify_shared_build.yml
@@ -87,7 +87,7 @@ jobs:
     name: Build macOS wheels
     needs:
     - test
-    runs-on: macos-14
+    runs-on: macos-13
 
     steps:
     - uses: actions/checkout@v4
@@ -96,6 +96,7 @@ jobs:
       uses: pypa/cibuildwheel@v2.23
       env:
         CIBW_ARCHS_MACOS: x86_64
+        MACOSX_DEPLOYEMENT_TARGET: 10.13
 
   macos-wheels-arm:
     name: Build macOS wheels for ARM (Native)

